support/ovscon2024/index.html

```diff
@@ -20,102 +20,560 @@
+    <ul>OVS + offloading engine offers following advantages:
+        <li>Flow-based actions such as RSS, VLAN stripping/insertion, packet mirroring, and multicasting. </li>
+        <li>Efficient IPsec Processing To leverage the inline IPsec capability of the Marvell SoC, the ovs-offload-engine intercepts SA/SP policies configured via strongSwan and offloads them to the hardware using DPDK rte_security APIs, enhancing OVS IPsec offloading capabilities.</li>
+        <li>Connection Tracking offloading: The ovs-offload-engine efficiently compensates for the lack of offloading support for connection tracking in OVS DPDK</li>
+        <li>Virtio Integration enables high-performance network throughput, achieving speeds of up to 100Gbps.</li>
+        <li>Optimized Service Function Chaining Efficiently steering traffic through a sequence of service functions, such as firewalls, load balancers, and intrusion detection systems.</li>
+        <li>Future Enhancements: NIX hardware supports hierarchical scheduling, shaping, coloring (up to five levels), and flexible packet formatting like LSO and checksum generation, enhancing OVS hardware offloading viz currently managed by software.</li>
+    </ul>
+    These performance gains highlight the substantial benefits of integrating Marvell hardware accelerators with OVS hardware offloading, demonstrating the potential for enhanced speed, efficiency, and scalability in modern network environments.
+</p>
 
-    <li>Performance measurements or approaches to improving performance.</li>
+<h3 id="t12">Inclusive Naming</h3>
+<p>Speaker(s): Simon Horman, Red Hat, Inc.</p>
+<p>
+    Language shapes our communication, and there are few things as important to a
+    community as communication. This also applies to Open Source communities like Open
+    vSwitch and OVN. Over time, for a variety of reasons, our community has normalised the
+    use of various terms that on examination are not inclusive, which is not in keeping with the
+    values of our community. This presentation will explore what steps OVS has taken towards
+    acknowledging this problem, using more inclusive language. And what further steps could
+    be taken.
+</p>
 
-    <li>End-user or service provider experiences with Open vSwitch.</li>
+<h3 id="t13">Improving megaflow cache performance in Open vSwitch with branch prediction</h3>
+<p>Speaker(s): Emil Stahl, KTH Royal Institute of Technology; Q&A portion hosted by Eelco Chaudron, Red Hat.</p>
+<p>
+    The evolution of software-defined networking has increasingly favored hardware implemen-
+tations for packet processing in data planes. Despite these advancements, optimizing the
+
+interaction between the data plane and the control plane, referred to as the slow path, re-
+mains a critical challenge. This interaction is poised to become a significant bottleneck in
+
+high-throughput networks.
```

Comment:
Some mid-sentence empty lines, not sure if this will be rendered well.  Probably will, but still looks strange.

Reply:
Should be rendered fine because the p tag will mostly ignore that whitespace.  I'll address it.